src/ui/Sort/Sort.ts

```diff
@@ -14,16 +14,19 @@ import { Component } from '../Base/Component';
 import { IComponentBindings } from '../Base/ComponentBindings';
 import { ComponentOptions } from '../Base/ComponentOptions';
 import { Initialization } from '../Base/Initialization';
-import { SortCriteria } from './SortCriteria';
-import { AccessibleButton } from '../../utils/AccessibleButton';
+import { SortCriteria, VALID_DIRECTION } from './SortCriteria';
+import { AccessibleButton, ArrowDirection } from '../../utils/AccessibleButton';
 import { l } from '../../strings/Strings';
+import { findIndex } from 'underscore';
 
 export interface ISortOptions {
   sortCriteria?: SortCriteria[];
   caption?: string;
 }
 /**
  * The `Sort` component renders a widget that the end user can interact with to select the criterion to use when sorting query results.
+ *
+ * To improve accessibility, it's recommended to group `Sort` components in a container with `role="radiogroup"`.
```

Comment:
@fbeaudoincoveo 

If there is no ancestor with `role="radiogroup"`, the `Sort` component itself will gain `role="radiogroup"`. Should this be documented outside of the code?

[JSUI-3056](https://coveord.atlassian.net/browse/JSUI-3056) will probably also introduce `aria-controls` as an attribute that needs to be defined alongside the `radiogroup` role.

Reply:
By default, I made it add the `radiogroup` role to the the `Sort` component.

There isn't a huge disadvantage to having the `radiogroup` on the `Sort` component, but there is a considerable advantage to having it on an ancestor of all `Sort` components: it represents in the accessibility tree that only one sort and one direction can be active at a time.

Reply:
I'm late to the party, but I will definitely update https://docs.coveo.com/en/2666/javascript-search-framework/adding-aria-landmarks to include that recommendation.